src/TelegramPoll.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace NotificationChannels\Telegram;
+
+use JsonSerializable;
+use NotificationChannels\Telegram\Traits\HasSharedLogic;
+
+/**
+ * Class TelegramPoll.
+ */
+class TelegramPoll implements JsonSerializable
+{
+    use HasSharedLogic;
+
+    /**
+     * @param  string  $question
+     *
+     * @return self
+     */
+    public static function create(string $question = ''): self
+    {
+        return new self($question);
+    }
+
+    /**
+     * Message constructor.
+     *
+     * @param  string  $question
+     */
+    public function __construct(string $question = '')
+    {
+        $this->question($question);
+    }
+
+    /**
+     * Poll question.
+     *
+     * @param  string  $question
+     *
+     * @return $this
+     */
+    public function question(string $question): self
+    {
+        $this->payload['question'] = $question;
+
+        return $this;
+    }
+
+    /**
+     * Poll options.
+     *
+     * @param  array  $options
+     *
+     * @return $this
+     */
+    public function options(array $options): self
```

Comment:
This conflicts with the shared logic trait [here](https://github.com/laravel-notification-channels/telegram/blob/master/src/Traits/HasSharedLogic.php#L120).

Reply:
Any rename suggestion for this!

Reply:
How about "Choices" ?

Reply:
Great, I'll rename it